Client Database System/Client Database System/Product and Services.cpp

```diff
@@ -8,13 +8,109 @@
 
 #include "Product and Services.hpp"
 #include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <string>
 #include <vector>
 
 using namespace std;
 
-void AddPrice(vector<int>& priceVector){
+Products_and_Services::Products_and_Services(){
+    product = "No product";
+    serviceDescription = "No service description";
+}
+
+void Product_and_Services::ReadFromFile(vector<string>& productVector, vector<string>& descriptionVector){
+    ifstream prodAndServIn;
+    prodAndServIn.open("ProductsAndServicesFile.txt");
+    
+    while(!inputStreamVariable.eof()){
```

Comment:
This was my fault, inputStreamVariable was supposed to be an example, but you can name it anything. In this case you named it prodAndServIn, so whenever you want to reference the .txt file you would use the defined variable.